components/dashboard/src/admin/TeamDetail.tsx

```diff
@@ -64,8 +103,43 @@ export default function TeamDetail(props: { team: Team }) {
                 </div>
             </div>
             <div className="flex mt-6">
-                {!team.markedDeleted && teamMembers && <Property name="Members">{teamMembers.length}</Property>}
-                {!team.markedDeleted && <Property name="BillingMode">{billingMode?.mode || "---"}</Property>}
+                {!team.markedDeleted && <Property name="Members">{teamMembers?.length || "?"}</Property>}
+                {!team.markedDeleted && <Property name="Billing Mode">{billingMode?.mode || "---"}</Property>}
+                <Property name="Billing Strategy" actions={[]}>
+                    <span>
+                        {costCenter?.billingStrategy === CostCenter_BillingStrategy.BILLING_STRATEGY_STRIPE
+                            ? "stripe"
+                            : "other"}
+                    </span>
+                </Property>
+            </div>
+            <div className="flex mt-6">
+                <Property name="Current Cycle" actions={[]}>
+                    <span>
+                        {dayjs(costCenter?.billingCycleStart).format("D MMM")} -{" "}
+                        {dayjs(costCenter?.nextBillingTime).format("D MMM")}
```

Comment:
**nitpick**: Minor restructure to match the date mentioned above in Sign Up date. Ideally, this would reflect the usage period as is from the billing page, like Oct 16, 2022 - Nov 16, 2022 but due to text wrapping on smaller screen let's keep the year out for now and revisit.

```suggestion
                        {dayjs(costCenter?.billingCycleStart).format("MMM D")} -{" "}
                        {dayjs(costCenter?.nextBillingTime).format("MMM D")}
```